sfaira/versions/metadata/base.py

```diff
@@ -347,17 +347,20 @@ def get_descendants(self, node: str) -> List[str]:
         node = self.convert_to_id(node)
         return list(networkx.descendants(self.graph, node))
 
-    def is_a(self, query: str, reference: str) -> bool:
+    def is_a(self, query: str, reference: str, convert_to_id=True) -> bool:
         """
         Checks if query node is reference node or an ancestor thereof.
 
         :param query: Query node name. Node ID or name.
         :param reference: Reference node name. Node ID or name.
+        :param convert_to_id: If True -> apply self.convert_to_id to query and reference input
         :return: If query node is reference node or an ancestor thereof.
         """
-        query = self.convert_to_id(query)
-        reference = self.convert_to_id(reference)
-        return query in self.get_ancestors(node=reference) or query == reference
+        if convert_to_id:
+            query = self.convert_to_id(query)
+            reference = self.convert_to_id(reference)
```

Comment:
This requires a statement that guarantees that the query is a list, otherwise `(reference in query)` will be buggy. Suggestion: Extend to:
```
if convert_to_id:
    query = self.convert_to_id(query)
    reference = self.convert_to_id(reference)
else:
    was_str = isinstance(query, str)
        if was_str:
            query = [query]
```

Alternatively refactor this "is-a-list" check into a separat function that is called here and within `convert_to_id`.

Reply:
ah, I missed that. Is fixed now!